### PR TITLE
2020 update

### PR DESCRIPTION
**Pull request by jterry64**
- Update to TCL 2020 in annualupdate and treecoverloss
- Add carbon layers to treecoverloss
- Remove carbonflux_minimal and annualupdate analyses that will no longer be used

### CODE REVIEW

**Review by thomas-maschler — APPROVED** [review]

File: src/main/scala/org/globalforestwatch/summarystats/annualupdate_minimal/AnnualUpdateMinimalSummary.scala [hunk 1]
```diff
@@ -109,15 +109,28 @@ object AnnualUpdateMinimalSummary {
...
         val co2Pixel = biomassPixel * co2Factor
-        //        val mangroveBiomassPixel = mangroveBiomass * areaHa
-        //        val mangroveCo2Pixel = mangroveBiomassPixel * co2Factor
+
+        val grossEmissionsCo2eNonCo2: Float = raster.tile.grossEmissionsCo2eNonCo2.getData(col, row)
+        val grossEmissionsCo2eCo2Only: Float =  raster.tile.grossEmissionsCo2eCo2Only.getData(col, row)
+        val grossCumulAbovegroundRemovalsCo2: Float =
+          raster.tile.grossCumulAbovegroundRemovalsCo2.getData(col, row)
+        val grossCumulBelowgroundRemovalsCo2: Float =
+          raster.tile.grossCumulAbovegroundRemovalsCo2.getData(col, row)
```
Comment (dagibbs22) [inline src/main/scala/org/globalforestwatch/summarystats/annualupdate_minimal/AnnualUpdateMinimalSummary.scala:123]: grossCumulBelowgroundRemovalsCo2 is using grossCumulAbovegroundRemovalsCo2. Correct to grossCumulBelowgroundRemovalsCo2. This is making the belowground removals have the same values as the aboveground removals.

**Review by dagibbs22 — CHANGES_REQUESTED** [review]
I found two mistakes: 1. the belowground removals uses the aboveground removals at one point (I put a comment there), 2. The output doesn't include "is__umd_tree_cover_gain_2000-2012" based on my use of the output tables. I see code for adding those output columns but they're not actually being output in change or summary-- not sure where the mistakes are.

File: src/main/scala/org/globalforestwatch/summarystats/annualupdate_minimal/AnnualUpdateMinimalSummary.scala (hunk 1 above)
Comment (jterry64) [reply src/main/scala/org/globalforestwatch/summarystats/annualupdate_minimal/AnnualUpdateMinimalSummary.scala:123]: good catch!

**Review by dagibbs22 — CHANGES_REQUESTED** [review]
I just ran the ArcPy client on PNG and NOR so I could compare their results to the carbonflux package. A few questions/corrections:
1. The ArcPy client package is outputting `umd_tree_cover_gain_2000-2013`, instead of 2000-2012. 
2. What is the column `sum((avg_whrc_aboveground_biomass_stock_2000__Mg_ha-1 * umd_tree_cover_extent_2000__ha))`?
3. Rename `avg_whrc_aboveground_biomass_stock_2000__Mg_ha-1` to `avg_whrc_aboveground_biomass_density_2000__Mg_ha-1`
4. `avg_whrc_aboveground_biomass_stock_2000__Mg_ha-1`  is not calculated correctly. I don't know exactly where that is calculated so I don't know where to point to. It should be the total AGB for that row divided by the total forest area for that row. If that's not possible to do, you can delete that field. 
5. Remove the `whrc_aboveground_co2_stock_2000__Mt` field.
6. Remove the `gfw_gross_emissions_co2e_all_gases_YEAR__Mg` and `whrc_aboveground_biomass_loss_YEAR__Mg` fields (per Slack conversation).

**Comment by dagibbs22** [comment]
> I just ran the ArcPy client on IDN and NOW so I could compare their results to the carbonflux package. A few questions/corrections:
> 
> 1. The ArcPy client package is outputting `umd_tree_cover_gain_2000-2013`, instead of 2000-2012.
> 2. The IDN statistics are incorrect but the Norway statistics are correct. For example, umd_tree_cover_extent_2000_ha is 42.9 Mha according to this run, which does not match the 139 Mha here: https://gfw.global/2Nvw7uf
> 3. What is the column `sum((avg_whrc_aboveground_biomass_stock_2000__Mg_ha-1 * umd_tree_cover_extent_2000__ha))`?
> 4. Rename `avg_whrc_aboveground_biomass_stock_2000__Mg_ha-1` to `avg_whrc_aboveground_biomass_density_2000__Mg_ha-1`
> 5. `avg_whrc_aboveground_biomass_stock_2000__Mg_ha-1`  is not calculated correctly. I don't know exactly where that is calculated so I don't know where to point to. It should be the total AGB for that row divided by the total forest area for that row. If that's not possible to do, you can delete that field.
> 6. Remove the `whrc_aboveground_co2_stock_2000__Mt` field.
> 7. Remove the `gfw_gross_emissions_co2e_all_gases_YEAR__Mg` and `whrc_aboveground_biomass_loss_YEAR__Mg` fields (per Slack conversation).

I reran this on my test areas of Norway and PNG with the correct JAR and the results matched what I expected. Field were correct. treecoverloss package used for the ArcPy client looks good now.

**Review by dagibbs22 — APPROVED** [review]

**Pull request merged**